force-app/main/default/classes/LeadTriggerHandler.cls

```diff
@@ -60,20 +62,22 @@ public with sharing class LeadTriggerHandler {
 	 * - If the lead belongs to the 'Technology' industry, increment score by another 10 points.
 	 */
 	public static void handleAutoLeadScoring(List<Lead> leadsToScore) {
+		System.debug('RUN: handleAutoLeadScoring');
+		System.debug('leadsToScore: ' + leadsToScore);
```

Comment:
I understand why these debugs are here and you might have left them on purpose for testing but remember when in production try to remove the debug statements